x11-libs/libva-intel-media-driver/metadata.xml

```diff
@@ -1,7 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<maintainer type="project">
-	<email>media-video@gentoo.org</email>
-</maintainer>
+	<maintainer type="project">
+		<email>media-video@gentoo.org</email>
+	</maintainer>
+	<maintainer type="project">
```

Comment:
You're not a project! I'll fix on merge.

Reply:
Who knows? I haven't asked my parents. ;)

I'll fix it with the next update.